app/models/manageiq/providers/container_manager.rb

```diff
@@ -22,6 +22,12 @@ class ContainerManager < BaseManager
     has_one :container_deployment, :foreign_key => :deployed_ems_id, :inverse_of => :deployed_ems
     has_many :computer_systems, :through => :container_nodes
 
+    # Archived entities to destroy when the container manager is deleted
+    has_many :old_container_groups, :foreign_key => :old_ems_id, :dependent => :destroy
```

Comment:
@zeari Maybe I am mistaken but it seems to me that you need to tell it the type of the class to use when it is not clear `:class_name => "ContainerGroups"`